src/wallets/IdentityWallet.ts

```diff
@@ -46,7 +50,7 @@ export class IdentityWallet implements TLWallet {
   private identityImplementationAddress: string
   private nonceMechanism: NonceMechanism
   private chainId: number
-  private identityVersion = 1
+  private metaTransactionVersion = 1
```

Comment:
Because it tracks the version of the meta-transaction protocol and should not be increased when updating the identity.
It is used on the identity contract upon verification of the signature: https://github.com/trustlines-protocol/contracts/blob/master/contracts/identity/Identity.sol#L269